ui/src/features/Editor/components/TopBar/components/AssetsDialog/AssetDeletionDialog.tsx

```diff
@@ -0,0 +1,30 @@
+import ConfirmationDialog from "@flow/features/ConfirmationDialog";
+import { useT } from "@flow/lib/i18n";
+
+type Props = {
+  assetToBeDeleted: string | undefined;
+  setAssetToBeDeleted: (asset?: string) => void;
+  onDeleteAsset: (id: string) => void;
+};
+
+const AssetDeletionDialog: React.FC<Props> = ({
+  assetToBeDeleted,
+  setAssetToBeDeleted,
+  onDeleteAsset,
+}) => {
+  const t = useT();
+  return (
+    <ConfirmationDialog
+      title={t("Are you absolutely sure?")}
+      description={t(
+        "This action cannot be undone. This will permanently delete your asset. Deletion of an asset could cause issues projects or deployments that are referencing this asset.",
```

Comment:
Please double check the language is clear enough

Reply:
Not sure if the text here was changed, but the grammar seems off.
Please use:

"This action cannot be undone. Deleting this asset will permanently remove it from your account and may cause issues with any projects or deployments that reference it. Please ensure this asset is no longer in use before proceeding."